code/model/Translatable.php

```diff
@@ -1155,6 +1155,11 @@ protected function addTranslatableFields(&$fields) {
 		// used in LeftAndMain->init() to set language state when reading/writing record
 		$fields->push(new HiddenField("Locale", "Locale", $this->owner->Locale));
 		
+		//ensures New DataObjects in a Gridfield have the correct Locale applied 
+		if($this->owner->ID == 0 && !($this->owner instanceof SiteTree)) {
```

Comment:
I'd assume this breaks if CMS is not installed? Why is it important if the owner is an instance of `SiteTree`?